xen-bugtool

```diff
@@ -1516,6 +1521,31 @@ def multipathd_topology(cap):
 
     return stdout
 
+def filter_snmp_xs_conf(_):
```

Comment:
parameter `_` is needless

Reply:
For `func_output`, it need an arguments `cap`, or else it will throw an exception.
```
Traceback (most recent call last):
  File "/usr/sbin/xen-bugtool", line 574, in collect_data
    s = v['func'](cap)
TypeError: filter_snmpd_conf() takes no arguments (1 given)
```

Reply:
If it's must have parameter, suggest to just keep there. 
Can be added in logging if you just want to avoid "unused parameter" like warning.

Reply:
[BengangY](https://github.com/BengangY) is righthttps://stackoverflow.com/questions/5787277/python-underscore-as-a-function-parameter:

> `_` is used to indicate that the input variable is a throwaway variable/parameter and thus might be required or expected, but will not be used in the code following it.

While probably not mentioned in a PEP, it's an industry standard among the Python IDEs and code checkers: https://github.com/PyCQA/pyflakes/issues/202